### PR TITLE
Allow validating consuming projects

### PR DESCRIPTION
**Pull request by skitt**
Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

File: .github/workflows/validate-consuming.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Checkout the Shipyard repository
+        uses: actions/checkout@master
+
+      - name: Reclaim free space!
+        run: |
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          df -h
+          free -h
+
```
Comment (mkolesnik) [inline .github/workflows/validate-consuming.yaml:27]:
```suggestion
```

This part is only needed for E2E, not necessary for validations/unit tests and such things

File: .github/workflows/validate-consuming.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Built the latest Shipyard image
```
Comment (mkolesnik) [inline .github/workflows/validate-consuming.yaml:28]:
```suggestion
      - name: Build the latest Shipyard image
```

File: .github/workflows/validate-consuming.yaml (hunk 1 above)
Comment (skitt) [reply .github/workflows/validate-consuming.yaml]: Removed, thanks!

File: .github/workflows/validate-consuming.yaml (hunk 2 above)
Comment (skitt) [reply .github/workflows/validate-consuming.yaml]: Fixed, and in `e2e` :-P

File: .github/workflows/e2e-consuming.yaml [hunk 3]
```diff
@@ -31,7 +31,7 @@ jobs:
           df -h
           free -h
 
-      - name: Built the latest Shipyard image
+      - name: Build the latest Shipyard image
```
Comment (mkolesnik) [inline .github/workflows/e2e-consuming.yaml:34]: :+1:

**Comment by skitt** [comment]
The failures are all Fedora mirror-related, so I’m merging this anyway — all the consuming projects passed.

**Pull request merged**